### ISSUE TITLE
Shortcuts to change app volume

### ISSUE THREAD

**Issue by HugoHeneault** [kyleneideck/BackgroundMusic#125]
It would be great to setup shortcuts to lower / up app volume :)

**Comment by kyleneideck** [comment]
Agreed, good idea. Hotkeys for the volume of the app in the foreground would be useful. Is that what you were thinking?

**Comment by HugoHeneault** [comment]
Yeah, or even some panel to define shortcuts for every app running. 
For example, I'm using Discord for speaking with teammates while working, but when no one is speaking, I'd like to put my chrome volume higher (and lower later), even if it's not the foreground app.

**Comment by mrboli** [comment]
+1!!! This is why I installed this app, in hopes of shortcuts for balancing the volumes!

**Comment by mrboli** [comment]
Is this being worked on already, @kyleneideck I can try to tackle this in a fork && pull request if not.

**Comment by kyleneideck** [comment]
@mrboli I haven't started on it. Happy for you to give it a go. Let me know if you get stuck or have any questions.

**Comment by attheshow** [comment]
I wonder if there would be any way to do this through the Apple Keyboard preferences pane?
